models.js

```diff
@@ -15,6 +15,8 @@ debuglog = function () {
 
 (function() {
     var xpm = xpathmodels;
+    xpm.validHashtagNamespaces = [];
+    xpm.hashtagToXPathDict = {};
```

Comment:
Vacillating on this, but leaning toward `XPathModels(hashtagInterpreter)` rather than having a `setHashtagInterpreter()` method. That way the resulting `xpathmodels` object can be treated as an immutable object, and it won't tempt people to do silly things like changing the interpreter on the global instance.